Packs/DomainToolsIrisDetect/Integrations/DomainToolsIrisDetect/DomainToolsIrisDetect.py

```diff
@@ -0,0 +1,1337 @@
+        create_common_api_arguments(args) | create_escalated_api_arguments(args),
+    )
+
+
+def domaintools_iris_detect_get_blocklist_domains_command(
+        client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    domaintools_iris_detect_get_blocklist_domains_command: Get the blocked domains list.
+    Args:
+        client: DomainTools client to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        fetch_domain_tools_api_results: A ``CommandResults`` object that is then passed
+         to ``return_results``, that contains result which will display in
+         war room.
+    """
+    return fetch_domain_tools_api_results(
+        client,
+        DOMAINTOOLS_WATCHED_DOMAINS_ENDPOINT,
+        DOMAINTOOLS_BLOCKED_DOMAINS_HEADER,
+        create_common_api_arguments(args) | create_escalated_api_arguments(args) | {"escalation_types[]": "blocked"},
+    )
+
+
+def domaintools_iris_detect_get_escalated_domains_command(
+        client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    domaintools_iris_detect_get_escalated_domains_command: Get the escalated domains
+    list.
+    Args:
+        client: DomainTools client to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        fetch_domain_tools_api_results: A ``CommandResults`` object that is then passed
+         to ``return_results``, that contains result which will display in
+         war room.
+    """
+    return fetch_domain_tools_api_results(
+        client,
+        DOMAINTOOLS_WATCHED_DOMAINS_ENDPOINT,
+        DOMAINTOOLS_ESCALATE_DOMAINS_HEADER,
+        create_common_api_arguments(args) | create_escalated_api_arguments(args) | {
+            "escalation_types[]": "google_safe"},
+    )
+
+
+def domaintools_iris_detect_get_monitors_list_command(
+        client: Client, args: Dict[str, Any]
+) -> CommandResults:
+    """
+    Get the monitor domains list.
+
+    Args:
+        client: DomainTools client to use.
+        args: Command arguments, usually passed from ``demisto.args()``.
+
+    Returns:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        which contains the result to display in the war room.
+    """
+
+    results, title = get_results_helper(
+        client,
+        DOMAINTOOLS_MONITOR_DOMAINS_ENDPOINT,
+        {
+            "datetime_counts_since": arg_to_datetime(args.get("datetime_counts_since")),
+        }
+        | create_common_api_arguments(args)
+        | create_escalated_api_arguments(args),
+        "monitors", DOMAINTOOLS_MONITORS_HEADER
+    )
+
+    return CommandResults(
+        outputs=results,
+        outputs_prefix=f"{INTEGRATION_CONTEXT_NAME}.Monitor",
+        outputs_key_field="",
+        readable_output=(
+            tableToMarkdown(
+                name=title,
+                t=[format_monitor_fields(result) for result in results],
```

Comment:
If you want to control the order of the table's headers then you can add the argument 'headers' to the tableToMarkdown command. In this parameter you can send to the function a list with the table headers in the order that you wish them to be. An example can be found in the attached picture.
![image](https://github.com/demisto/content/assets/71597826/d1125c81-32d4-4ef9-8e75-e21503141a68)